linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -335,40 +335,40 @@ where
 
 #[test(tokio::test)]
 async fn test_memory_handle_block_proposal_zero_amount() {
-    let client = MemoryStoreClient::make_test_client(None).await;
-    run_test_handle_block_proposal_zero_amount(client).await;
+    let store = MemoryStoreClient::make_test_client(None).await;
+    run_test_handle_block_proposal_zero_amount(store).await;
 }
 
 #[cfg(feature = "rocksdb")]
 #[test(tokio::test)]
 async fn test_rocks_db_handle_block_proposal_zero_amount() {
     let _lock = ROCKS_DB_SEMAPHORE.acquire().await;
-    let client = RocksDbStoreClient::make_test_client(None).await;
-    run_test_handle_block_proposal_zero_amount(client).await;
+    let store = RocksDbStoreClient::make_test_client(None).await;
+    run_test_handle_block_proposal_zero_amount(store).await;
 }
 
 #[cfg(feature = "aws")]
 #[test(tokio::test)]
 async fn test_dynamo_db_handle_block_proposal_zero_amount() {
-    let client = DynamoDbStoreClient::make_test_client(None).await;
-    run_test_handle_block_proposal_zero_amount(client).await;
+    let store = DynamoDbStoreClient::make_test_client(None).await;
```

Comment:
Perhaps we should continue in a next PR and write `let store = DynamoDbStore::make_test_store`.
Also we can decide that "Store = storage client" and use storage for core implementations.

Reply:
@MathieuDutSik What do you think?

Reply:
We also use both `store` and `storage`. I'd go with the former everywhere, since that's what the trait is called, and it's shorter.